### ISSUE TITLE
Error when passing a null value to a nullable Json field type

### ISSUE THREAD

**Issue by radicand** [prisma/prisma-client-js#693]
## Bug description

When attempting to insert a null value for a nullable Json field, an error is thrown in Prisma Client

I haven't dived into this in great depth, but on https://github.com/prisma/prisma/blob/master/src/packages/client/src/runtime/query.ts#L684 , obj.values is accessed before obj is checked for null; perhaps the root cause.

## How to reproduce

1. Create a model with a nullable Json field (e.g., `Json?`)
2. Insert a null
3. Crash

```
// For the below object to be created against the schema below:
debug: generation options: {"metric":"metric_two","details":null}
error: TypeError: Cannot read property 'values' of null
    at stringify (.../node_modules/@prisma/client/src/runtime/query.ts:684:13)
    at Arg._toString (.../node_modules/@prisma/client/src/runtime/query.ts:772:23)
    at Arg.toString (.../node_modules/@prisma/client/src/runtime/query.ts:781:17)
...
```

## Expected behavior

Null to be inserted

## Prisma information

Sanitized schema:
```
model metrics {
  details                   Json?
  id                        Int                        @default(autoincrement()) @id
  metric                    metrics_metric?
  createdAt                 DateTime                   @default(now())
  updatedAt                 DateTime                   @updatedAt

  @@index([metric], name: "metrics_metric")
}
enum metrics_metric {
  metric_one
  metric_two @map("metric-two")
  three
}
```

## Environment & setup

- OS: Mac OS
- Database: MySQL 5.7
- Prisma version: 2.0.0-alpha.1198
- Node.js version: v13.13.0

**Comment by pantharshit00** [comment]
Thanks, I can confirm the issue. The query doesn't even reach the engine it is an error in our client code as you already pointed out.

The query I used along with the model that you provided in order to reproduce this issue:
```ts
  const data = await client.metrics.create({
    data: {
      metric: null,
      details: null,
    },
  });
```

![image](https://user-images.githubusercontent.com/22195362/82122950-4e319080-97b4-11ea-9a9b-cb19199f655d.png)

**Comment by Jolg42** [comment]
Thank you for the issue!

This is now fixed in the latest alpha (2.0.0-alpha.1263).
